sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/KnownMethodId.cs

```diff
@@ -39,5 +39,33 @@ public static class KnownMethodId
         /// The instructions with this method ID are removed from UCFG.
         /// </summary>
         public static readonly string Unknown = "__unknown";
+
+        /// <summary>
+        /// The method ID that the security engine uses for attributes/annotations. It accepts
+        /// one argument and returns one value. The argument should be the result of the invocation
+        /// of the attribute constructor. The returned value should be assigned to a variable
+        /// with the same name as the parameter this attribute is applied on.
+        /// </summary>
+        /// <example>
+        /// public void Foo([Bar]string s) { ... }
+        ///
+        /// Instructions:
+        /// %0 = Bar()
+        /// s = __annotation(%0)
+        /// ...
+        /// </example>
+        public static readonly string Annotation = "__annotation";
```

Comment:
I know this feedback come late but I strongly think we should have these constants defined in the `.proto` file itself instead of redefining them in all languages. This would allow to avoid typo and sync of values.

Reply:
I think this feedback should be done in a different place (if adding such constants in protobuf is possible at all).

Reply:
My 2 cents: Let's create a technical ticket for that and not block this PR. Why? Because it's going to require a PR on sonar-ucfg and so it's not a small change